alpha-blog/app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     if @user.save
       session[:user_id] = @user.id
       flash[:notice] = "Welcome to Alpha Blog #{@user.username}, you have successfully signed up!"
-      redirect_to articles_path
+      redirect_to root_path
```

Comment:
Why this change in this PR? 
This PR says delete users. This is create o

Reply:
ooo wanted to change to another path😅